### DISCUSSION TITLE
Protest Finder/ Call For Protest

### DISCUSSION THREAD

**Discussion by AnonIranOne** [WeAreMahsaAmini/Core#330] [Ideas]
One of the biggest problems of people in different cities is finding each other to start their protests. Finding each other in the middle of the regime's soldiers is scary and too risky.

We need to develop a platform to people can create a "call" for a protest. Also, other people can see protest calls in their city.

Features:
- every call can be public or sent to just trusted friends/people on the platform
- someone can report a person az MOZDUR
- in every call, protesters can decide about a 'sign' to find each other. For example, a blue shirt, or red bracelet or ...
- everyone can confirm if they participate in the protest or not to estimate the number of participants.
- people can vote for a protest up or down. In the case of parallel protest, each call that has a higher score is the main on a specific date in a particular city
- after protest every protest can have its own census. Every participant could announce that is ok and not arrested. They could plan for their next protest. THE MOST THING THAT CAN CLOSE US TO WINNING IS TO KEEP GOING AND REPEAT PROTESTS.

Don't know how:
- we need to find a way to send notifications to others: SMS, push notif, call ...
- find a way to find and block infiltrators

**Comment by forwomanlifefreedom** [comment]
سلام بنظرم ایده خوبیه و مشابه ایده‌ایه که قبلا اینجا #491 گفتم ولی تمرکزم بیشتر روی اعتصابات بود. فقط مشکل اساسی که این ایده‌ها دارن اینه که چطور باید افرادو احراز هویت کرد که اون فرد سایبری و مزدور و.. نباشه؟ درواقع احراز هویت که میشه کرد ولی خب کسی اعتماد نمیکنه که با هویت واقعی خودش وارد سیستم بشه.
